src/rprblender/nodes/__init__.py

```diff
@@ -103,7 +105,8 @@ def get_current_convert_nodes():
         NodeItem('ShaderNodeBrightContrast'),
         NodeItem('ShaderNodeGamma'),
         NodeItem('ShaderNodeInvert'),
-        NodeItem('ShaderNodeMixRGB'),
+        RPR_NodeItem("ShaderNodeMix", label="Mix Color", settings={"data_type": "'RGBA'"}, blender_version=">=3.4"),
```

Comment:
We can change it to:
`NodeItem("ShaderNodeMix", label="Mix Color", settings={"data_type": "'RGBA'"}, poll=lambda cls: BLENDER_VERSION >= "3.4")`

Reply:
Thanks, improved within https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/pull/582/commits/5a251cc6f55f589967f41bfc55ab22951c98bdaa